docs/training-run/index.txt

```diff
@@ -4,106 +4,255 @@
  Run Training Code
 ###################
 
-In this document, we will cover how to run your training code. You can submit your code to a cluster
-and run them as experiments. We will go over the whole cycle of it.
+This section covers how to run training code by submitting your code to a cluster and running it as
+an experiment.
```

Comment:
I think we're missing a preamble that explains the doc at the very highest level.  I think that preamble should explain:
- your `entrypoint` is either the legacy bare-Trial-class form, or it's a command with args that we run in the container (that's so simple!).
- For distributed training, it's often wise to separate a "launcher" that kicks off some number of distributed workers from your training script (which is typically run on each worker).
- You can write your own custom launcher, or use one of our predefined launchers.